core/src/test/java/com/linecorp/armeria/common/stream/TimeoutStreamMessageTest.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+import com.linecorp.armeria.common.TimeoutException;
+
+import io.netty.util.concurrent.DefaultEventExecutor;
+import io.netty.util.concurrent.EventExecutor;
+
+class TimeoutStreamMessageTest {
+    private EventExecutor executor;
+
+    @BeforeEach
+    public void setUp() {
+        executor = new DefaultEventExecutor();
+    }
+
+    @AfterEach
+    public void tearDown() {
+        executor.shutdownGracefully();
+    }
```

Comment:
We can use 
```suggestion

    @RegisterExtension
    static final EventLoopExtension executor = new EventLoopExtension();
```
Please use `executor.get()` when using it.